src/aarch64.cpp

```diff
@@ -0,0 +1,340 @@
+/*
+This is a part of Loops project.
+Distributed under Apache 2 license.
+See https://github.com/vpisarev/loops/LICENSE
+*/
+
+#include "aarch64.hpp"
+#include "composer.hpp"
+#include "func_impl.hpp"
+#include <unordered_map>
+
+namespace loops
+{
+
+inline Arg arg_hidden_reg_constructor(size_t reg) //TODO(ch): What the hell? Create normal walk class and make these functions method of class.
+{
+    Arg res;
+    res.tag = Arg::IREG;
+    res.idx = reg;
+    return res;
+}
+
+m2b_map instrucion_set;
+void init_instrucion_set()
+
+{
+    { //LDRSW:
+        //TODO(ch): Ldrsw also have many options to be added here. Literal loading, register shift or post/pre-indexing. Format isn't full now and hardcoded.
+        binatr ldrswr({
```

Comment:
Frankly speaking, I don't like this table-driven encoding scheme at all.
I think, aarch64 encoder for this instruction subset can fit into 300 lines of code, maybe less, without generic backend and composer classes. And it will have much cleaner structure.

But maybe I'm wrong. Please, provide implementation for the same instruction subset for x64. Then it will become more clear whether this design is flexible enough or not.